macros/GroupData.json

```diff
@@ -1199,19 +1199,33 @@
         },
         "Payment Request API": {
             "overview":   [ "Payment Request API" ],
-            "interfaces": [ "PaymentAddress",
+            "interfaces": [ "MerchantValidationEvent",
+                            "PaymentAddress",
+                            "PaymentMethodChangeEvent",
                             "PaymentRequest",
                             "PaymentRequestUpdateEvent",
-                            "PaymentResponse",
-                            "BasicCardRequest",
-                            "BasicCardResponse" ],
+                            "PaymentResponse" ],
+            "dictionaries": [ "BasicCardRequest",
+                            "BasicCardResponse",
+                            "PaymentMethodData",
+                            "PaymentCurrencyAmount",
+                            "PaymentDetailsBase",
+                            "PaymentDetailsModifier",
+                            "PaymentDetailsUpdate",
+                            "PaymentItem",
+                            "PaymentOptions",
+                            "PaymentShippingOption",
+                            "PaymentShippingType" ],
             "methods":    [],
             "properties": ["HTMLIFrameElement.allowPaymentRequest",
                            "ServiceWorkerGlobalScope.onabortpayment",
```

Comment:
I also got this request from Marcos a little while ago:

"hey, when you can, could you delete:

* ServiceWorkerGlobalScope.onabortpayment
* ServiceWorkerGlobalScope.oncanmakepayment
* ServiceWorkerGlobalScope.onpaymentrequest
* ServiceWorkerRegistration.paymentManager

from the Payment Request sidebar? Those should eventually go into the Payment Handler docs, but they are not part of Payment Request